app/assets/stylesheets/resource_curriculum_show.scss

```diff
@@ -1,15 +1,15 @@
 @import 'color';
 
-a {
-  .resources_container & {
-    color: $brick;
-  }
-  .resources_container &:hover {
+
+.resources_container a, .curriculum_container a {
```

Comment:
Selector `resources_container` should be written in lowercase with hyphens<br>Selector `curriculum_container` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line